### PR TITLE
Support Python 3.5 in Parquet code

### PR DESCRIPTION
**Pull request by quasiben**
- [ ] Tests added / passed
- [ ] Passes `black dask` / `flake8 dask`

PR is an alternate of https://github.com/dask/dask/pull/5490 -- this requires black from src

### CODE REVIEW

**Comment by mrocklin** [comment]
@quasiben you may not have installeed black from git master in this PR

**Comment by quasiben** [comment]
@mrocklin forgot to push the big changelog

File: .travis.yml [hunk 1]
```diff
@@ -75,7 +75,7 @@ script:
   - if [[ $TEST_HDFS == 'true' ]]; then source continuous_integration/hdfs/run_tests.sh; fi
   - if [[ $TEST == 'true' ]]; then source continuous_integration/travis/run_tests.sh; fi
   - if [[ $LINT == 'true' ]]; then pip install flake8 ; flake8 dask; fi
-  - if [[ $LINT == 'true' ]]; then pip install black ; black dask --check; fi
+  - if [[ $LINT == 'true' ]]; then pip install pip install git+https://github.com/psf/black; black dask --check; fi
```
Comment (mrocklin) [inline .travis.yml]:
```suggestion
  - if [[ $LINT == 'true' ]]; then pip install pip install git+https://github.com/psf/black@73bd7038fbefdb1c6a61fa1edf16ff61613050a5; black dask --check; fi
```

Comment (mrocklin) [reply .travis.yml]: Can we also include this commit in the `.pre-commit.yaml` file?

**Comment by mrocklin** [comment]
+1 from me.  I'll plan to merge soon if there are no objections.

**Comment by mrocklin** [comment]
Supercedes #5490

**Comment by mrocklin** [comment]
Thanks for tracking this down @quasiben !

**Pull request merged**